### PR TITLE
fix(pie-monorepo): DSW-1969 updated Playwright and Percy dependencies

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
## Describe your changes (can list changeset entries if preferable)
- Updated Percy and Playwright dependencies
- Updated tests to support updated Playwright deps

A couple things to note with this PR is that all screenshots added are now covering the Chrome browser.
Some differences in the screenshots of existing packages are due to the change of order in some of the listed variations.

Another note is there is 1 test that has been skipped. This seems to be a legitimate failure of the component test that is blocking this work so a ticket will be written to address this.

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 - @jamieomaguire & @ashleynolan 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: configs/pie-components-config/playwright-lit-visual-config.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ export function getPlaywrightVisualConfig () {
             /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
             trace: 'on',
             testIdAttribute: 'data-test-id',
+            discovery: {
+                disallowedHostnames: [
```
Comment (xander-marjoram) [inline configs/pie-components-config/playwright-lit-visual-config.js:34]: Do you have a link to the docs that explain this?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -45,6 +45,9 @@ env:
   PERCY_TOKEN_PIE_TEXTAREA: ${{ secrets.PERCY_TOKEN_PIE_TEXTAREA }}
   PERCY_TOKEN_PIE_TOAST: ${{ secrets.PERCY_TOKEN_PIE_TOAST }}
   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  PERCY_DO_NOT_CAPTURE_RESPONSIVE_ASSETS: true
```
Comment (xander-marjoram) [inline .github/workflows/ci.yml:48]: Why did you need to add this?

File: configs/pie-components-config/playwright-lit-visual-config.js (hunk 1 above)
Comment (JoshuaNg2332) [reply configs/pie-components-config/playwright-lit-visual-config.js:34]: https://www.browserstack.com/docs/percy/common-issue/disallowing-requests

File: .github/workflows/ci.yml (hunk 2 above)
Comment (JoshuaNg2332) [reply .github/workflows/ci.yml:48]: It may have been causing some of the request issues we were seeing in some of our builds and was suggested by the Percy team to add it in as part of the troubleshooting steps.

File: configs/pie-components-config/playwright-lit-visual-config.js (hunk 1 above)
Comment (xander-marjoram) [reply configs/pie-components-config/playwright-lit-visual-config.js:34]: Interesting, I'm assuming it still fetches the assets just doesn't rely on the asset discovery process. I wonder if we'll see fewer script failures but more diffs with assets that failed to load.

**Review by ashleynolan — APPROVED** [review]

**Review by jamieomaguire — APPROVED** [review]
